aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/strategy/LogErrorContextMissingStrategy.java

```diff
@@ -25,15 +25,15 @@
     private static final Log logger = LogFactory.getLog(LogErrorContextMissingStrategy .class);
 
     /**
-     * Logs {@code message} on the {#code error} level.
+     * Logs {@code message} on the {@code error} level, and a stacktrace at {@code debug} level.
      * @param message the message to log
      * @param exceptionClass the type of exception suppressed in favor of logging {@code message}
      */
     @Override
     public void contextMissing(String message, Class<? extends RuntimeException> exceptionClass) {
         logger.error("Suppressing AWS X-Ray context missing exception (" + exceptionClass.getSimpleName() + "): " + message);
         if (logger.isDebugEnabled()) {
-            logger.debug(new RuntimeException(message));
+            logger.debug("", new RuntimeException(message));
```

Comment:
```suggestion
            logger.debug("Attempted to find context at:", new RuntimeException(message));
```

May as well have something instead of empty string